src/test/java/com/hivemq/cli/commands/hivemq/script/ScriptCreateCommandTest.java

```diff
@@ -0,0 +1,161 @@
+/*
+ * Copyright 2019-present HiveMQ and the HiveMQ Community
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hivemq.cli.commands.hivemq.script;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.hivemq.cli.commands.hivemq.datahub.OutputFormatter;
+import com.hivemq.cli.openapi.ApiException;
+import com.hivemq.cli.openapi.JSON;
+import com.hivemq.cli.openapi.hivemq.DataHubScriptsApi;
+import com.hivemq.cli.openapi.hivemq.Script;
+import com.hivemq.cli.rest.HiveMQRestService;
+import com.hivemq.cli.utils.TestLoggerUtils;
+import com.hivemq.cli.utils.json.ScriptSerializer;
+import org.jetbrains.annotations.NotNull;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import picocli.CommandLine;
+
+import java.io.ByteArrayOutputStream;
+import java.io.File;
+import java.io.IOException;
+import java.io.PrintStream;
+import java.nio.file.Files;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyDouble;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.spy;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+public class ScriptCreateCommandTest {
+
+    private final @NotNull HiveMQRestService hiveMQRestService = mock(HiveMQRestService.class);
```

Comment:
```suggestion
    private final @NotNull HiveMQRestService hiveMQRestService = mock();
```

You could omit the class inside the parameter. It is inferred over the return type. We started to remove them in our other projects.